app/main.py

```diff
@@ -2,0 +1,20 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format. Use: mv <source> <destination>")
+    source, destination = parts[1], parts[2]
+    if not os.path.isfile(source):
+        raise FileNotFoundError(f"Source file '{source}' does not exist.")
+    if destination.endswith("/"):
+        destination = os.path.join(destination, os.path.basename(source))
+    destination_dir = os.path.dirname(destination)
+    if destination_dir and not os.path.exists(destination_dir):
+        os.makedirs(destination_dir)
+    with open(source, "r") as src_file:
+        with open(destination, "w") as dest_file:
```

Comment:
When opening files, it's a good practice to specify the encoding, especially when dealing with text files. Consider using `open(source, 'r', encoding='utf-8')` and `open(destination, 'w', encoding='utf-8')` to avoid potential encoding issues.